--- origsrc/soprano-2.0.99/CMakeLists.txt	2008-07-08 18:46:59.000000000 -0500
+++ src/soprano-2.0.99/CMakeLists.txt	2008-07-20 18:20:34.515625000 -0500
@@ -43,7 +43,7 @@
 
 
 if(CMAKE_COMPILER_IS_GNUCXX AND NOT WIN32 )
-    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wnon-virtual-dtor -Wno-long-long -ansi -Wundef -Wcast-align -Wchar-subscripts -Wall -W -Wpointer-arith -Wformat-security -fno-check-new -fno-common")
+    set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wnon-virtual-dtor -Wno-long-long -Wundef -Wcast-align -Wchar-subscripts -Wall -W -Wpointer-arith -Wformat-security -fno-check-new -fno-common")
 endif(CMAKE_COMPILER_IS_GNUCXX AND NOT WIN32 )
 if(MSVC)
     add_definitions(-D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE -Zc:wchar_t-)
