--- origsrc/soprano-2.2.3/backends/redland/CMakeLists.txt	2009-02-02 13:29:29.000000000 -0600
+++ src/soprano-2.2.3/backends/redland/CMakeLists.txt	2009-05-05 22:37:56.093625000 -0500
@@ -49,7 +49,7 @@ if(BUILD_REDLAND_BACKEND)
 
   qt4_automoc(${redland_backend_SRC})
 
-  add_library(soprano_redlandbackend SHARED ${redland_backend_SRC})
+  add_library(soprano_redlandbackend MODULE ${redland_backend_SRC})
 
   TARGET_LINK_LIBRARIES(soprano_redlandbackend soprano_redlandcore)
 
@@ -59,8 +59,6 @@ if(BUILD_REDLAND_BACKEND)
 
   INSTALL(TARGETS soprano_redlandbackend
     LIBRARY DESTINATION ${LIB_DESTINATION}/soprano
-    RUNTIME DESTINATION bin/soprano
-    ARCHIVE DESTINATION lib/soprano
     )
 
   configure_file(redlandbackend.desktop.cmake ${CMAKE_CURRENT_BINARY_DIR}/redlandbackend.desktop)
--- origsrc/soprano-2.0.99/parsers/nquads/CMakeLists.txt	2007-11-23 03:56:23.000000000 -0600
+++ src/soprano-2.0.99/parsers/nquads/CMakeLists.txt	2008-07-20 19:11:46.703125000 -0500
@@ -13,14 +13,12 @@
 
 qt4_automoc(${nquadparser_SRC})
 
-add_library(soprano_nquadparser SHARED ${nquadparser_SRC})
+add_library(soprano_nquadparser MODULE ${nquadparser_SRC})
 
 TARGET_LINK_LIBRARIES(soprano_nquadparser soprano)
 
 INSTALL(TARGETS soprano_nquadparser
 	LIBRARY DESTINATION ${LIB_DESTINATION}/soprano
-	RUNTIME DESTINATION bin/soprano
-	ARCHIVE DESTINATION lib/soprano
 )
 
 configure_file(nquadparser.desktop.cmake ${CMAKE_CURRENT_BINARY_DIR}/nquadparser.desktop)
--- origsrc/soprano-2.0.99/parsers/raptor/CMakeLists.txt	2007-10-15 13:23:31.000000000 -0500
+++ src/soprano-2.0.99/parsers/raptor/CMakeLists.txt	2008-07-20 19:11:57.875000000 -0500
@@ -17,7 +17,7 @@
 
 qt4_automoc(${raptor_parser_SRC})
 
-add_library(soprano_raptorparser SHARED ${raptor_parser_SRC})
+add_library(soprano_raptorparser MODULE ${raptor_parser_SRC})
 
 TARGET_LINK_LIBRARIES(soprano_raptorparser soprano ${RAPTOR_LIBRARIES})
 
@@ -27,8 +27,6 @@
 
 INSTALL(TARGETS soprano_raptorparser
 	LIBRARY DESTINATION ${LIB_DESTINATION}/soprano
-	RUNTIME DESTINATION bin/soprano
-	ARCHIVE DESTINATION lib/soprano
 )
 
 configure_file(raptorparser.desktop.cmake ${CMAKE_CURRENT_BINARY_DIR}/raptorparser.desktop)
--- origsrc/soprano-2.0.99/queryparsers/rasqal/CMakeLists.txt	2007-10-01 10:45:15.000000000 -0500
+++ src/soprano-2.0.99/queryparsers/rasqal/CMakeLists.txt	2008-07-20 19:03:55.828125000 -0500
@@ -20,7 +20,7 @@
 
 qt4_automoc(${rasqalqueryparser_SRC})
 
-add_library(soprano_rasqalqueryparser SHARED ${rasqalqueryparser_SRC})
+add_library(soprano_rasqalqueryparser MODULE ${rasqalqueryparser_SRC})
 
 TARGET_LINK_LIBRARIES(soprano_rasqalqueryparser soprano ${RAPTOR_LIBRARIES} ${RASQAL_LIBRARIES} ${REDLAND_LIBRARIES})
 
@@ -32,8 +32,6 @@
 
 INSTALL(TARGETS soprano_rasqalqueryparser
 	LIBRARY DESTINATION ${LIB_DESTINATION}/soprano
-	RUNTIME DESTINATION bin/soprano
-	ARCHIVE DESTINATION lib/soprano
 )
 
 configure_file(rasqalqueryparser.desktop.cmake ${CMAKE_CURRENT_BINARY_DIR}/rasqalqueryparser.desktop)
--- origsrc/soprano-2.0.99/serializers/nquads/CMakeLists.txt	2007-11-23 03:56:23.000000000 -0600
+++ src/soprano-2.0.99/serializers/nquads/CMakeLists.txt	2008-07-20 19:04:17.093750000 -0500
@@ -13,14 +13,12 @@
 
 qt4_automoc(${nquadserializer_SRC})
 
-add_library(soprano_nquadserializer SHARED ${nquadserializer_SRC})
+add_library(soprano_nquadserializer MODULE ${nquadserializer_SRC})
 
 TARGET_LINK_LIBRARIES(soprano_nquadserializer soprano)
 
 INSTALL(TARGETS soprano_nquadserializer
 	LIBRARY DESTINATION ${LIB_DESTINATION}/soprano
-	RUNTIME DESTINATION bin/soprano
-	ARCHIVE DESTINATION lib/soprano
 )
 
 configure_file(nquadserializer.desktop.cmake ${CMAKE_CURRENT_BINARY_DIR}/nquadserializer.desktop)
--- origsrc/soprano-2.0.99/serializers/raptor/CMakeLists.txt	2007-10-06 09:24:30.000000000 -0500
+++ src/soprano-2.0.99/serializers/raptor/CMakeLists.txt	2008-07-20 19:04:25.531250000 -0500
@@ -16,7 +16,7 @@
 
 qt4_automoc(${raptor_serializer_SRC})
 
-add_library(soprano_raptorserializer SHARED ${raptor_serializer_SRC})
+add_library(soprano_raptorserializer MODULE ${raptor_serializer_SRC})
 
 TARGET_LINK_LIBRARIES(soprano_raptorserializer soprano soprano_redlandcore ${RAPTOR_LIBRARIES})
 
@@ -28,8 +28,6 @@
 
 INSTALL(TARGETS soprano_raptorserializer
 	LIBRARY DESTINATION ${LIB_DESTINATION}/soprano
-	RUNTIME DESTINATION bin/soprano
-	ARCHIVE DESTINATION lib/soprano
 )
 
 configure_file(raptorserializer.desktop.cmake ${CMAKE_CURRENT_BINARY_DIR}/raptorserializer.desktop)
